packages/token-list/_tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+	// Currently disabled, package does not typecheck
```

Comment:
Token list had it's types checked before, but doesn't type check now. Worth checking what's changed in the config in this PR. I did use a more strict set of rules.

Reply:
Not sure if the plan is to sort this out before merge, but I'd rather remove the type-checking altogether than to include the temporary TODO file.

Reply:
> I did use a more strict set of rules.

Noticing specifically a change in `noImplicitAny`, which I recall to have been the subject of previous discussion at https://github.com/WordPress/gutenberg/pull/17014#discussion_r317923126 .